packaging/Dockerfile.fedora-nmstate-dev

```diff
@@ -2,7 +2,7 @@
 # Fedora official repository
 # (https://hub.docker.com/r/fedora/systemd-systemd/).
 # It enables systemd to be operational.
-FROM docker.io/library/fedora:31
```

Comment:
32 is also available as tag, please use it instead for now. It is possibly a little bit more stable.

Reply:
Thanks! Will do.

Reply:
Done!